### ISSUE TITLE
Muse Auxiliary Electrode source

### ISSUE THREAD

**Issue by ElliotMebane** [NeuroTechX/eeg-notebooks_v0.1#26]
Here's a place to buy Muse auxiliary gold-cup electrodes to use with this project. You may like to include the link below in your docs where you describe the cable you used in your experiments. Avoid the time and hassle of making a cable yourself! 
https://docs.google.com/document/d/1CGsimrSyLjyH7mXBcm0LEOw83m-nEqd6ImoSHwTtoJM

**Comment by JohnGriffiths** [comment]
Thanks for the info Elliot, that's really cool. 

I'll add this info to the docs. 

Are you making recordings with these?

**Comment by ElliotMebane** [comment]
I originally made them as part of a NeuroTechX workshop: NeuroTechLA | Hands-on Muse / BCI Workshop.  We used Muse-LSL and did some first steps in a Machine Learning app that was able to distinguish between 2 different brainwave patterns. One of the pictures in the Google Doc I linked-to shows the electrode in use as a 5th channel in Muse-LSL. Workshop file link is below, and also linked-to in my doc. https://github.com/NeuroTechX/bci-workshop/blob/master/INSTRUCTIONS.md
